app/views/hub/metrics/index.html.erb

```diff
@@ -21,79 +21,172 @@
     </div>
   </div>
 
-  <div>
+  <div class="scrollable-table-wrapper">
```

Comment:
I feel like there is an org or users table somewhere that's not being wrapped with this class, and we should probably do that.

Reply:
it's the users table